### PR TITLE
CONSOLE-3678: Convert EventStream component in events.jsx from class …

### PR DESCRIPTION
**Pull request by tvu20**
…component to functional component

EventStream converted from a class component to a functional component in order to use react-router v6 hooks

### CODE REVIEW

File: frontend/public/components/events.jsx [hunk 1]
```diff
@@ -359,210 +358,152 @@ class EventStream extends React.Component {
...
-    let sysEventStatus, statusBtnTxt;
-
-    if (noEvents || mock || (noMatches && resourceEventStream)) {
-      sysEventStatus = <NoEvents />;
-    }
-    if (noMatches && !resourceEventStream) {
-      sysEventStatus = <NoMatchingEvents allCount={allCount} />;
-    }
+  const toggleStream = () => {
+    setActive((prev) => !prev);
```
Comment (jhadvig) [inline frontend/public/components/events.jsx:429]: Love how the conversion is actually cleaning up the code 🙌

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

Nice work @tvu20 👍

**Comment by jhadvig** [comment]
This PR is techdebt cleanup and has no user-facing impact, so I am manually assigning docs and px approvals.

/assign @yapei for QE approval.
/label docs-approved
/label px-approved

**Comment by XiyunZhao** [comment]
This PR has been tested on private build, with manual and regression testing, and no issue was found
/label qe-approved

**Pull request merged**